ynput/OpenPype
# Maya: Xgen fixes

## Changelog Description
Fix for Xgen extraction of world parented nodes and validation for required namespace.

## Testing notes:
1. Setup Xgen on geometry without namespace that parent to the world.
2. Publish should fail.
3. Add namespace to Xgen and geometry.
4. Publish should succeed.
